packages/storybook-utils/src/mandatory.ts

```diff
@@ -0,0 +1,44 @@
+import { type Decorator } from "@storybook/vue3";
+import { ref, watch } from "vue";
+
+type MandatoryIndicator = "required" | "optional";
+type MandatoryGlobalType = {
```

Comment:
This is a general Storybook type, right? We could make it more generic by adding a generic for the items value so we can re-use this type int he future for additional globals

Reply:
done